SwinjectAutoregistration.xcodeproj/project.pbxproj

```diff
@@ -874,6 +877,7 @@
 				);
 				INFOPLIST_FILE = "$(SRCROOT)/Sources/Info.plist";
 				PRODUCT_BUNDLE_IDENTIFIER = "com.swinject.SwinjectAutoregistration-iOS";
+				SWIFT_VERSION = 5.0;
```

Comment:
This would force consumers of the app to run against Swift 5, should we actually do that?

Reply:
That's a fair point. We should probably keep it at its current value.